tests/baselines/reference/for-of32.errors.txt

```diff
@@ -0,0 +1,7 @@
+tests/cases/conformance/es6/for-ofStatements/for-of32.ts(1,10): error TS7022: 'v' implicitly has type 'any' because it is does not have a type annotation and is referenced directly or indirectly in its own initializer.
+
+
+==== tests/cases/conformance/es6/for-ofStatements/for-of32.ts (1 errors) ====
+    for (var v of v) { }

```

Comment:
I would assume it means that `v` as declared in the `for-of` statement is not usable from within the expression, so an outer-defined `v` would not be shadowed (and thus, still usable) at that point.